attendispeechservice/src/main/kotlin/nl/attendi/attendispeechservice/components/attendimicrophone/microphone/AttendiMicrophoneViewModel.kt

```diff
@@ -28,12 +26,12 @@ import nl.attendi.attendispeechservice.components.attendirecorder.recorder.Atten
  * and plugin management to this ViewModel.
  *
  * @param recorder An [AttendiRecorder] instance that handles low-level audio recording operations.
- * @param plugins A list of [AttendiMicrophonePlugin] instances to be activated and managed alongside the microphone lifecycle.
  */
 class AttendiMicrophoneViewModel(
     private val recorder: AttendiRecorder,
-    private val plugins: List<AttendiMicrophonePlugin>,
-    private val onMicrophoneTapCallback: suspend () -> Unit
+    private val microphoneSettings: AttendiMicrophoneSettings,
+    private val onMicrophoneTapCallback: () -> Unit,
```

Comment:
I think the `Callback` is redundant / self-evident?
```suggestion
    private val onMicrophoneTap: () -> Unit,
```